### ISSUE TITLE
Unable to install with strange error: file 'C:\Users\JMatos\AppData\Local\Temp\pip-build-or8r1snd\memory-profiler\mprof.bat' does not exist

### ISSUE THREAD

**Issue by jcrmatos** [pythonprofilers/memory_profiler#178]
Hello,

I'm trying to install memory_profiler (using pip 9.0.1) under a Python 3.5.4 32b venv (psutil 5.4.1. is already installed) on a Windows 7 Pro SP1 x64, but it returns the following error:

```
(dexp) C:\Users\JMatos\OneDrive\Python\dexp\dexp>pip install memory_profiler
Collecting memory_profiler
  Using cached memory_profiler-0.48.0.tar.gz
Requirement already satisfied: psutil in c:\users\jmatos\envs\dexp\lib\site-pack
ages (from memory_profiler)
Building wheels for collected packages: memory-profiler
  Running setup.py bdist_wheel for memory-profiler ... error
  Complete output from command c:\users\jmatos\envs\dexp\scripts\python.exe -u -
c "import setuptools, tokenize;__file__='C:\\Users\\JMatos\\AppData\\Local\\Temp
\\pip-build-or8r1snd\\memory-profiler\\setup.py';f=getattr(tokenize, 'open', ope
n)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __
file__, 'exec'))" bdist_wheel -d C:\Users\JMatos\AppData\Local\Temp\tmp7ho7msbap
ip-wheel- --python-tag cp35:
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build\lib
  copying memory_profiler.py -> build\lib
  running build_scripts
  creating build\scripts-3.5
  copying and adjusting mprof -> build\scripts-3.5
  error: file 'C:\Users\JMatos\AppData\Local\Temp\pip-build-or8r1snd\memory-prof
iler\mprof.bat' does not exist

  ----------------------------------------
  Failed building wheel for memory-profiler
  Running setup.py clean for memory-profiler
Failed to build memory-profiler
Installing collected packages: memory-profiler
  Running setup.py install for memory-profiler ... error
    Complete output from command c:\users\jmatos\envs\dexp\scripts\python.exe -u
 -c "import setuptools, tokenize;__file__='C:\\Users\\JMatos\\AppData\\Local\\Te
mp\\pip-build-or8r1snd\\memory-profiler\\setup.py';f=getattr(tokenize, 'open', o
pen)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code,
__file__, 'exec'))" install --record C:\Users\JMatos\AppData\Local\Temp\pip-n4n3
4xfy-record\install-record.txt --single-version-externally-managed --compile --i
nstall-headers c:\users\jmatos\envs\dexp\include\site\python3.5\memory-profiler:

    running install
    running build
    running build_py
    creating build
    creating build\lib
    copying memory_profiler.py -> build\lib
    running build_scripts
    creating build\scripts-3.5
    copying and adjusting mprof -> build\scripts-3.5
    error: file 'C:\Users\JMatos\AppData\Local\Temp\pip-build-or8r1snd\memory-pr
ofiler\mprof.bat' does not exist

    ----------------------------------------
Command "c:\users\jmatos\envs\dexp\scripts\python.exe -u -c "import setuptools,
tokenize;__file__='C:\\Users\\JMatos\\AppData\\Local\\Temp\\pip-build-or8r1snd\\
memory-profiler\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.re
ad().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" inst
all --record C:\Users\JMatos\AppData\Local\Temp\pip-n4n34xfy-record\install-reco
rd.txt --single-version-externally-managed --compile --install-headers c:\users\
jmatos\envs\dexp\include\site\python3.5\memory-profiler" failed with error code
1 in C:\Users\JMatos\AppData\Local\Temp\pip-build-or8r1snd\memory-profiler\

(dexp) C:\Users\JMatos\OneDrive\Python\dexp\dexp>
```

Thanks for the help in advance,

JM

**Comment by fabianp** [comment]
Should be fixed in 7e02ef4ea8a6138297f1c08d26f10dc98cd6f4b5 and I just made a new release with this fix. Could you do again `pip install memory_profiler` and let us know if the problem is fixed?

**Comment by jcrmatos** [comment]
Hello,

It's fixed.

Thanks,

JM
